src/_utils/radio/index.unit.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import Checkbox from './index'
+
+describe('Checkbox', () => {
```

Comment:
nit: I think i'll be better to test explicitely what should be rendered, something like:

```
const radio = shallow(<Radio isChedked />)
expect (radio.find(CircleIcon).toExist()).toBe(true)
```

Reply:
Snapshots are fine for html-rendering checks. I like both solutions, so, dunno, I'll do a poll to get others opinions ^^